datasource/timeouts/schema_test.go

```diff
@@ -17,6 +17,72 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework-timeouts/internal/validators"
 )
 
+func TestBlockWithOpts(t *testing.T) {
+	t.Parallel()
+
+	type testCase struct {
+		opts     timeouts.Opts
+		expected schema.Block
+	}
+	tests := map[string]testCase{
+		"empty-opts": {
+			opts: timeouts.Opts{},
+			expected: schema.SingleNestedBlock{
+				CustomType: timeouts.Type{
+					ObjectType: types.ObjectType{
+						AttrTypes: map[string]attr.Type{
+							"read": types.StringType,
+						},
+					},
+				},
+				Attributes: map[string]schema.Attribute{
+					"read": schema.StringAttribute{
+						Optional: true,
+						Validators: []validator.String{
+							validators.TimeDuration(),
+						},
+					},
```

Comment:
Hm, wouldn't this also have the default description or am I thinking about this wrong?